.clang-format

```diff
@@ -0,0 +1,109 @@
+# .clang-format
+#
+# Alignment
+# ---------
+#
+# Align:
+# - fonction parameters on the open bracket,
+# - trailing comments.
+# Don't align:
+# - consecutive assignments or declarations,
+# - operands of binary and ternary expressions.
+#
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignOperands: false
+AlignTrailingComments: true
+#
+# Short statements
+# ----------------
+#
+# Forbid short
+# - blocks
+# - case labels
+# - functions
+# - if statements
+# - loops
+# on a single line.
+#
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: false
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+#
+# Line breaks
+# -----------
+#
+# Insert line breaks:
+# - before braces (Allman style),
+# - after the function definition return type,
```

Comment:
I prefer to have the return type in line if we have space (clang Penalty.. stuff for the 80col autofill)